src/rdo.rs

```diff
@@ -240,41 +242,51 @@ pub fn rdo_mode_decision(
     };
 
     // Find the best chroma prediction mode for the current luma prediction mode
-    let cfl = &CFLParams::new();
     for &chroma_mode in &mode_set_chroma {
+      let mut cfl_alpha_set = vec![ [-1, 0], [-1, 1], [-1, -1], [1, -1], [0, -1], [1, 0], [0, 1], [1, 1] ];
+      if chroma_mode != PredictionMode::UV_CFL_PRED {
+        cfl_alpha_set.truncate(1);
```

Comment:
Here we avoid iterating CfL parameters for modes other than `UV_CFL_PRED` by truncating the list to a single placeholder.

Reply:
Yes, but it iterated once for other modes, for ex chroma DC_PRED?
Is that intended?

Reply:
I tried cfl_alpha_set.truncate(0); then it skips the alpha search loop.

Reply:
Yes, I've initially misunderstood that the design where parameter search is located. It is put inner most of current nested rdo loops. So, setting cfl to the 1st entry of cfl_alpha_set[] and passing to encode_block_b() seems harmless for non CFL_PRED modes. Wouldn't it be helpful if cfl parameter to encode_block_b() does not have any info(like null or None) when choma mode is mot CFL_REPD?